python/llm/example/GPU/HF-Transformers-AutoModels/Model/chatglm3/README.md

```diff
@@ -56,44 +61,51 @@ export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
 <summary>For Intel Data Center GPU Max Series</summary>
 
 ```bash
+# Recommended Environment Variables for optimal performance
 export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libtcmalloc.so
 export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+export SYCL_CACHE_PERSISTENT=1
 export ENABLE_SDP_FUSION=1
 ```
-> Note: Please note that `libtcmalloc.so` can be installed by `conda install -c conda-forge -y gperftools=2.10`.
+> [!NOTE]
+> Please note that `libtcmalloc.so` can be installed by `conda install -c conda-forge -y gperftools=2.10`.
```

Comment:
It seems that `> [!NOTE]` does not work inside `<details>`

For notebox inside `<details>`, let's keep this as before.